site/en/docs/devtools/javascript/background-services/index.md

```diff
@@ -20,14 +23,18 @@ DevTools considers each of the following APIs to be a background service:
 - [Notifications][4]
 - [Push Messages][5]
 
-Chrome DevTools can log background service events for 3 days, even when DevTools is not open. This
-can help you make sure that events are being sent and received as expected. You can also inspect the
-details of each event.
+Chrome DevTools can log fetch, sync, and notification events for 3 days, even when DevTools is not open. This can help you make sure that events are being sent and received as expected.
```

Comment:
```suggestion
Chrome DevTools can log fetch, sync, and notification events for three days, even when DevTools is not open. This can help you make sure that events are being sent and received as expected.
```

Reply:
I think we probably want to spell out three here, based on https://developers.google.com/style/numbers